pkg/ingester/ingester_v2.go

```diff
@@ -413,6 +419,7 @@ func (i *Ingester) getOrCreateTSDB(userID string, force bool) (*tsdb.DB, error)
 	}
 
 	i.TSDBState.dbs[userID] = db
+	memUsers.Inc()
```

Comment:
This is better handled in https://github.com/cortexproject/cortex/pull/1982

Reply:
Right. I will rebase and remove this.

Reply:
Aah, #1982 is not yet merged. I was wondering why I don't see conflicts yet :)

Reply:
Removed from this PR now.

Reply:
Please remind to remove mentioning `cortex_ingester_memory_users` from changelog too.